php/libraries/NDB_Config.class.inc

```diff
@@ -570,8 +570,11 @@ class NDB_Config
             $nextLevel = NDB_Config::getMenuTabs($thisRow['ID']);
 
             if (!empty($nextLevel)) {
-                if (NDB_Config::checkMenuPermission($nextLevel[0]['ID'])) {
-                    $thisRow['subtabs'] = $nextLevel;
+                foreach ($nextLevel as $k=>$info) {
```

Comment:
@alisterdev I usually use the same format everywhere for consistancy but I'm certain it could be omitted